src/components/card/Card.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import classes from "./Card.module.css";
+
+const Card = ({ icon, image, title, details }) => {
+  return (
+    <div className={classes.container}>
```

Comment:
They should all be placed inside dev and given a container class

Reply:
i will do that